sdk/cosmos/microsoft-azure-cosmos-benchmark/pom.xml

```diff
@@ -8,12 +8,12 @@ Licensed under the MIT License.
   <parent>
     <groupId>com.microsoft.azure</groupId>
     <artifactId>azure-cosmos-parent</artifactId>
-    <version>3.4.0</version> <!-- {x-version-update;com.microsoft.azure:azure-cosmos-parent;current} -->
+    <version>3.5.0</version> <!-- {x-version-update;com.microsoft.azure:azure-cosmos-parent;current} -->
```

Comment:
Why are we skipping one version here to go to 3.5.0 from 3.4.0 ?

Reply:
com.microsoft.azure:azure-cosmos-parent already has a 3.4.0 release, I just ticked this up to the next major.

Reply:
This is related to the previous question and will be answered above.